engines/python/src/main/java/ai/djl/python/engine/PyModel.java

```diff
@@ -167,7 +168,7 @@ public void load(Path modelPath, String prefix, Map<String, ?> options) throws I
                 } else if ("nc".equals(manager.getDevice().getDeviceType())
                         && pyEnv.getTensorParallelDegree() > 0) {
                     entryPoint = "djl_python.transformers_neuronx";
-                } else if (isTrtLlmBackend) {
+                } else if ("trtllm".equals(features)) {
```

Comment:
why we cannot use the old param?

Reply:
Because that param checks whether rolling_batch == trtllm. Might as well, remove the param. 

```
case "rolling_batch":
      isTrtLlmBackend = "trtllm".equals(value);
       break;
 ```    
                    
https://github.com/deepjavalibrary/djl-serving/pull/1678/files#diff-244860c398daaa2ff4bc328b98c867ccce93643b1f65c4f1024bf6bdd5a7beecR124